<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CavityControlWindow</class>
 <widget class="QMainWindow" name="CavityControlWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QDockWidget" name="jpe_cpshr3_dockWidget">
      <property name="minimumSize">
       <size>
        <width>398</width>
        <height>561</height>
       </size>
      </property>
      <property name="windowTitle">
       <string>JPE CPSHR3 control</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents_8">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_34">
          <property name="text">
           <string>Temperature [K]</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="jpe_temperature_spinBox">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>293</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>293</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_38">
          <property name="text">
           <string>Frequency [Hz]</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="jpe_frequency_spinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>600</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>600</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_39">
          <property name="text">
           <string>Step size [%]</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="jpe_step_size_spinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>Tracking</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_27">
             <item>
              <widget class="QLabel" name="label_33">
               <property name="text">
                <string>X [µm]</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="x_tracking_doubleSpinBox">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>-1000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="set_x_pushButton">
               <property name="text">
                <string>Set X</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="reset_x_pushButton">
               <property name="text">
                <string>Reset X</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_26">
             <item>
              <widget class="QLabel" name="label_32">
               <property name="text">
                <string>Y [µm]</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="y_tracking_doubleSpinBox">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>-1000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="set_y_pushButton">
               <property name="text">
                <string>Set Y</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="reset_y_pushButton">
               <property name="text">
                <string>Reset Y</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_31">
               <property name="text">
                <string>Z [µm]</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="z_tracking_doubleSpinBox">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>-1000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="set_z_pushButton">
               <property name="text">
                <string>Set Z</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="reset_z_pushButton">
               <property name="text">
                <string>Reset Z</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>Control</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="3" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="step_size_label">
               <property name="text">
                <string>Displacement [µm]</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="displacement_doubleSpinBox">
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="stop_positionner_pushButton">
               <property name="text">
                <string>ABORD !</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QGroupBox" name="CLA1_groupBox">
               <property name="title">
                <string>CLA 1</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_13">
                <item row="0" column="0">
                 <widget class="QPushButton" name="cla1_step_up_pushButton">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="text">
                   <string>^</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QPushButton" name="cla1_step_down_pushButton">
                  <property name="text">
                   <string>v</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="CLA2_groupBox">
               <property name="title">
                <string>CLA 2</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_14">
                <item row="1" column="0">
                 <widget class="QPushButton" name="cla2_step_down_pushButton">
                  <property name="text">
                   <string>v</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QPushButton" name="cla2_step_up_pushButton">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="text">
                   <string>^</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="CLA3_groupBox">
               <property name="title">
                <string>CLA 3</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_15">
                <item row="1" column="0">
                 <widget class="QPushButton" name="cla3_step_down_pushButton">
                  <property name="text">
                   <string>v</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QPushButton" name="cla3_step_up_pushButton">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="text">
                   <string>^</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QGroupBox" name="XY_manual_positionning_groupBox">
               <property name="title">
                <string>X Y</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_10">
                <item row="0" column="1">
                 <widget class="QPushButton" name="step_yplus_pushButton">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="text">
                   <string>^</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QPushButton" name="step_xminus_pushButton">
                  <property name="text">
                   <string>&lt;</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QPushButton" name="step_xplus_pushButton">
                  <property name="text">
                   <string>&gt;</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QPushButton" name="step_yminus_pushButton">
                  <property name="text">
                   <string>v</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="Z_manual_positionning_groupBox">
               <property name="title">
                <string>Z</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_11">
                <item row="0" column="0">
                 <widget class="QPushButton" name="step_zplus_pushButton">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="text">
                   <string>^</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QPushButton" name="step_zminus_pushButton">
                  <property name="text">
                   <string>v</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="lock_checkBox">
             <property name="text">
              <string>Lock</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QDockWidget" name="scanner_widget">
      <property name="minimumSize">
       <size>
        <width>390</width>
        <height>247</height>
       </size>
      </property>
      <property name="floating">
       <bool>false</bool>
      </property>
      <property name="windowTitle">
       <string>Scanner control</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents_6">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QRadioButton" name="scanner_ramp_radioButton">
          <property name="text">
           <string>Ramp</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Freq [Hz]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="scanner_ramp_frequency_DoubleSpinBox">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>20000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_35">
          <item>
           <widget class="QLabel" name="label_41">
            <property name="text">
             <string>Amp [V]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="scanner_sinewave_amplitude_DoubleSpinBox">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-10.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Offset [V]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="scanner_ramp_offset_DoubleSpinBox">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-10.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_36">
          <item>
           <widget class="QLabel" name="label_42">
            <property name="text">
             <string>Offset [V]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="scanner_sinewave_offset_DoubleSpinBox">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-10.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_25">
          <item>
           <widget class="QLabel" name="label_40">
            <property name="text">
             <string>Freq [Hz]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="scanner_sinewave_frequency_DoubleSpinBox">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>20000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <widget class="QRadioButton" name="scanner_dc_radioButton">
          <property name="text">
           <string>DC</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Amp [V]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="scanner_ramp_amplitude_DoubleSpinBox">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-10.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="scanner_sinewave_radioButton">
          <property name="text">
           <string>Sinewave</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="scanner_start_PushButton">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="scanner_stop_PushButton">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_38">
          <item>
           <widget class="QLabel" name="label_44">
            <property name="text">
             <string>Offset [V]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="scanner_dc_offset_DoubleSpinBox">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-10.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>416</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
